cmp/grid/ag-grid/styles.scss

```diff
@@ -74,17 +74,23 @@
     }
   }
 
-  // Generate indentations for tree grids.
-  // Based upon https://github.com/ag-grid/ag-grid/blob/master/packages/ag-grid-community/src/styles/ag-grid.scss#L1018
-  $xh-tree-indent: 10px !default;
-  @for $i from 1 to 50 {
-    .ag-ltr .ag-row-group-indent-#{$i} {
-      padding-left: $i * $xh-tree-indent;
+  &.xh-grid-tree-mode {
```

Comment:
*totally* minor tweak, but can this style be renamed to 'xh-grid-hierarchical' or 'xh-grid-indent-mode' or something like that?

I would expect from the current name, that these are styles that are always present when the model is treeMode= true  and that is not the case.